packages/react-native-sdk/src/components/call/internal/CallParticipantsSpotlight.tsx

```diff
@@ -60,13 +60,10 @@ export const CallParticipantsSpotlight = () => {
 const styles = StyleSheet.create({
   container: { flex: 1 },
   participantView: {
-    flex: 2,
-    overflow: 'hidden',
-    borderRadius: theme.rounded.sm,
+    flex: 1,
     marginHorizontal: theme.padding.sm,
   },
   participantVideoContainer: {
-    flex: 1,
     paddingVertical: theme.padding.sm,
   },
```

Comment:
this style seems like a big change, does it look the same or figma aligned now?

Reply:
Yes(from last test) it should be the same, may be if you can confirm after testing if something is broken. These were kinda unnecessary styles.

Reply:
Yeah, there were some issues that went unnoticed due to this change and I thought them to be working. They are solved now. ✅